Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/Rendering/BocColumnRendererBase.cs

```diff
@@ -339,13 +346,20 @@ void IBocColumnRenderer.RenderDataCell (BocColumnRenderingContext renderingConte
       renderingContext.Writer.AddAttribute(HtmlTextWriterAttribute2.Role, ariaRoleForTableDataElement);
       if (_renderingFeatures.EnableDiagnosticMetadata)
         AddDiagnosticMetadataAttributes(renderingContext);
+
+      var bocCellAttributeRenderingContext = new BocCellAttributeRenderingContext<TBocColumnDefinition>(renderingContext);
+      AddAttributesToRenderForDataCell(bocCellAttributeRenderingContext, in arguments);
       renderingContext.Writer.RenderBeginTag(HtmlTextWriterTag.Td);
 
       RenderCellContents(renderingContext, arguments);
 
       renderingContext.Writer.RenderEndTag();
     }
 
+    public virtual void AddAttributesToRenderForDataCell (BocCellAttributeRenderingContext<TBocColumnDefinition> renderingContext, in BocDataCellRenderArguments arguments)
```

Comment:
New API should be protected. Reason: `AddAttributesToRenderForDataCell` and `AddAttributesToRenderForTitleCell`  are meant as template methods and are only called within the class hierarchy.